genai/generativelanguagepb_veneer.gen.go

```diff
@@ -698,26 +700,34 @@ type GenerationConfig struct {
 	TopP *float32
 	// Optional. The maximum number of tokens to consider when sampling.
 	//
-	// The model uses combined Top-k and nucleus sampling.
-	//
+	// Models use nucleus sampling or combined Top-k and nucleus sampling.
 	// Top-k sampling considers the set of `top_k` most probable tokens.
+	// Models running with nucleus sampling don't allow top_k setting.
 	//
 	// Note: The default value varies by model, see the `Model.top_k`
-	// attribute of the `Model` returned from the `getModel` function.
+	// attribute of the `Model` returned from the `getModel` function. Empty
+	// `top_k` field in `Model` indicates the model doesn't apply top-k sampling
+	// and doesn't allow setting `top_k` on requests.
 	TopK *int32
+	// Optional. Output response mimetype of the generated candidate text.
+	// Supported mimetype:
+	// `text/plain`: (default) Text output.
+	// `application/json`: JSON response in the candidates.
+	ResponseMimeType string
```

Comment:
Spell as `ResponseMIMEType`. We like to have acronyms and other initialisms use the same case.

Reply:
updated, by just hand editing the generated file.
All I could find on protoveneer, as far as docs, was the config.go from that project and it didn't seem to support field renaming.
Happy to change the config if y'all could point me towards docs on how.

Reply:
Sorry, I should have told you how. 
There are no docs for protoveneer except in the code. :(
Here is a similar case in the config: https://github.com/google/generative-ai-go/blob/main/genai/config.yaml#L67

Reply:
nevermind, spoke too soon, figured out how the config works.  Just pushed a more proper fix